src/Plugin.php

```diff
@@ -25,6 +26,17 @@
  */
 class Plugin implements PluginEntryPointInterface
 {
+    /**
+     * @return string[]
+     */
+    protected function getFormStubs(): array
+    {
+        $forms = glob(__DIR__.'/Stubs/common/forms/*.stubphp') ?: [];
+        $events = glob(__DIR__.'/Stubs/common/forms/Event/*.stubphp') ?: [];
+
+        return array_merge($forms, $events);
+    }
+
```

Comment:
this method (getFormStubs) is not needed.
Changing the `getCommonStubs` as following should do the work:

```php
    protected function getCommonStubs(): array
    {
        return glob(__DIR__.'/Stubs/common/{*,*/*}.stubphp', GLOB_BRACE) ?: [];
    }
```